updating cwlogs collection docs (#3658)
SumoLogic/sumologic-documentation: docs/send-data/collect-from-other-data-sources/amazon-cloudwatch-logs/collect-with-cloudformation-template.md

After few minutes you will see `CREATE_COMPLETE` in the Status column.

:::note

> This solution only ingests log messages to sumo logic. If you want to include log levels (such as INFO, DEBUG,etc) to sumo logic, please find and update the regex in [lambda code](https://github.com/SumoLogic/sumologic-aws-lambda/blob/main/cloudwatchlogs-with-dlq/cloudwatchlogs_lambda.js#L14) deployed in your AWS account with 

Reviewer: ```suggestion
This solution only ingests log messages to Sumo Logic. If you want to include log levels (such as INFO, DEBUG, etc.) to Sumo Logic, find and update the regex in [Lambda code](https://github.com/SumoLogic/sumologic-aws-lambda/blob/main/cloudwatchlogs-with-dlq/cloudwatchlogs_lambda.js#L14) deployed in your AWS account with 
```

    <br/><img src={useBaseUrl('img/send-data/create-destination.png')} style={{border: '1px solid gray'}} alt="create-destination" width="700"/>
1. (Optional) In the **Test Pattern** section, select the log data to test, then click **Test pattern**. If test results look fine, then click **Start Streaming**.<br/><img src={useBaseUrl('img/send-data/test-pattern.png')} style={{border: '1px solid gray'}} alt="test-pattern" width="700"/>

:::note

> Sample cloudWatch Log ingested for Amazon RDS postgreSQL using Lambda Log forwarder with ```IncludeLogGroupInfo``` set to ```false``` . 

Reviewer: ```suggestion
Sample CloudWatch log ingested for Amazon RDS PostgreSQL using Lambda log forwarder with ```IncludeLogGroupInfo``` set to ```false``` . 
```

> Sample cloudWatch Log ingested for Amazon RDS postgresSQL using Lambda Log forwarder with ```IncludeLogGroupInfo``` set to ```false``` . 

Reviewer: ```suggestion
Sample cloudWatch Log ingested for Amazon RDS postgreSQL using Lambda Log forwarder with ```IncludeLogGroupInfo``` set to ```false``` . 
```
